unicodetools/org/unicode/propstest/PrintUnicodeSet.java

```diff
@@ -0,0 +1,14 @@
+package org.unicode.propstest;
```

Comment:
A quick and dirty tool for debugging because the unicode utilities are not yet functioning.